docs/sql/decimal.md

```diff
@@ -28,6 +28,36 @@ The legal operations are ``+`` (plus, unary and binary), ``-`` (minus,
 unary and binary), ``*`` (multiplication), ``/`` (division), ``%``
 (modulus).
 
+Modulus happens as follows: 
+<table>
```

Comment:
This table is much less helpful than the other one.
I would rather have a formula, I assume it's this one:

if x >= 0 and y > 0 then x - floor(x / y) * y
if x >= 0 and y < 0  then x % abs(y)
if x < 0 and y > 0 then - abs(x) % y
if x < 0 and y > 0 then - abs(x) % abs(y)

Reply:
this formula is probably the same for integer modulus, so we can put it there too

Reply:
Yeah, I think that formula is correct. I'll add it to the docs.